controllers/chat/group/changeChannelDetail.js

```diff
@@ -38,38 +41,63 @@ const changeChannelDetail = async (req, res) => {
       const textTargetUser = `${ownUser.username} changed the group name to "${channel_name}"`;
       const textDefaultUser = `${ownUser.username} changed the group name to "${channel_name}"`;
 
-      await channel.sendMessage({
-        text: textDefaultUser,
-        own_text: textOwnUser,
-        other_text: textTargetUser,
-        better_type: 'change_channel_detail',
-        type: 'system',
-        user_id: userId,
-        only_to_user_show: userId,
-        disable_to_user: false,
-        is_from_prepopulated: true,
-        system_user: userId,
-        isSystem: true,
-        members: members
-      });
+      await channel.sendMessage(
+        {
+          text: textDefaultUser,
+          own_text: textOwnUser,
+          other_text: textTargetUser,
+          better_type: 'change_channel_detail',
+          type: 'system',
+          user_id: userId,
+          only_to_user_show: userId,
+          disable_to_user: false,
+          is_from_prepopulated: true,
+          system_user: userId,
+          isSystem: true,
+          members: members
+        },
+        {
+          skip_push: true
+        }
+      );
+
+      const other_members = all_members.filter((m) => m !== userId);
+      await Promise.all(
+        other_members.map(async (m) => {
+          await BetterSocialCore.fcmToken.sendGroupChatNotification(m, textDefaultUser);
+        })
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here involve sending a message with additional options (`skip_push: true`) and sending notifications to other members. Ensure that the `channel.sendMessage` function supports these additional options. Also, make sure that the `BetterSocialCore.fcmToken.sendGroupChatNotification` function handles errors properly, as it's being called multiple times in a `Promise.all`. If one promise fails, it will reject the entire `Promise.all`.
```diff
+ await channel.sendMessage(
+   {
+     text: textDefaultUser,
+     own_text: textOwnUser,
+     other_text: textTargetUser,
+     better_type: 'change_channel_detail',
+     type: 'system',
+     user_id: userId,
+     only_to_user_show: userId,
+     disable_to_user: false,
+     is_from_prepopulated: true,
+     system_user: userId,
+     isSystem: true,
+     members: members
+   },
+   {
+     skip_push: true
+   }
+ );
+ 
+ const other_members = all_members.filter((m) => m !== userId);
+ await Promise.all(
+   other_members.map(async (m) => {
+     await BetterSocialCore.fcmToken.sendGroupChatNotification(m, textDefaultUser);
+   })
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->